src/ErrorHandler/ErrorHandler.php

```diff
@@ -67,7 +67,7 @@ public function handleCaughtThrowable(\Throwable $t, ThrowableRendererInterface
             $this->log($t);
             return $this->exposeDetails ? $renderer->renderVerbose($t) : $renderer->render($t);
         } catch (\Throwable $t) {
-            return nl2br((string)$t);
+            return (string)$t;
```

Comment:
This one fixes error rendering fallback. Previously it was just giving out 500 w/o any clues.